projects/angular/src/button/button-loading/loading-button.ts

```diff
@@ -76,20 +76,15 @@ export class ClrLoadingButton implements LoadingListener {
 
     switch (state) {
       case ClrLoadingState.DEFAULT:
-        this.renderer.removeStyle(this.el.nativeElement, 'width');
```

Comment:
Not sure how these style removals are solving the issue. The solution we talked about is that when a button is in the loading, keep its width at least at the spinner's width (it shouldn't be narrower than that). When it switches to the non-loading state, its width should expand to fit asynchronously loaded content.